solr/solr-ref-guide/src/meta-docs/publish.adoc

```diff
@@ -47,61 +47,26 @@ To build the HTML:
 [source,bash]
 $ ant clean default
 +
-This will produce pages with a DRAFT watermark across them. While these are fine for initial DRAFT publication, see the section <<Publish the Guide>> for steps to produce final production-ready HTML pages.
+This will produce pages with a DRAFT watermark across them. While these are fine for initial DRAFT publication, see the section <<Publish the Guide without watermark>> for steps to produce final production-ready HTML pages.
 . The resulting Guide will be in `solr/build/solr-ref-guide`. The HTML files themselves will be in `solr/build/solr-ref-guide/html-site`.
```

Comment:
Since you changed the heading this refers to back to its original, this reference should be changed too. This is why precommit failed.